### PR TITLE
Simple device binding additions

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 6d66b3f932c5cb953b3b3b8148722fb60411c4a9: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        6 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 7f566f6424af156c4ad06f7b62032217d712d6cc: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 1]
```diff
@@ -1238,15 +1238,71 @@ Before we describe the usable identifiers, let's quickly discuss how they can be
...
+    ...
+
+    // The key pair can also be obtained from the Android Keystore any time as follows:
+    KeyStore keyStore = KeyStore.getInstance("AndroidKeyStore");
+    keyStore.load(null);
+    PrivateKey privateKey = (PrivateKey) keyStore.getKey("key1", null);
+    PublicKey publicKey = keyStore.getCertificate("key1").getPublicKey();
+    ```
+
+  >Source: <https://developer.android.com/reference/android/security/keystore/KeyGenParameterSpec.html>
```
Comment (sushi2k) [inline Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md]: Maybe put that into the first line of the code.  It looks a bit lost here.

File: Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 2]
```diff
@@ -1238,15 +1238,71 @@ Before we describe the usable identifiers, let's quickly discuss how they can be
...
+    SecretKey key = keyGenerator.generateKey();
+
+    // The key can also be obtained from the Android Keystore any time as follows:
+    KeyStore keyStore = KeyStore.getInstance("AndroidKeyStore");
+    keyStore.load(null);
+    key = (SecretKey) keyStore.getKey("key2", null);
+    ```
 
-  Note: For API level 23 and above, the *KeyGenParameterSpec* API can be leveraged directly to generate and manage secret keys through the Android keystore.
+  >Source: <https://developer.android.com/reference/android/security/keystore/KeyGenParameterSpec.html>
```
Comment (sushi2k) [inline Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md]: Same as above

File: Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md [hunk 3]
```diff
@@ -1238,15 +1238,71 @@ Before we describe the usable identifiers, let's quickly discuss how they can be
...
+
+  - Encrypt the authentication data and other sensitive data stored by the application using secret key through AES-GCM cipher and use device specific parameters such as Instance ID, etc. as associated data
```
Comment (sushi2k) [inline Document/0x05j-Testing-Resiliency-Against-Reverse-Engineering.md]: secret key -> a secret key

**Comment by OMTGreleaser** [comment]
Results for commit f87226c9d247cf7b378831bd97ad0dfbcbca7f90: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        4 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**